### ISSUE TITLE
Where is the test set, please?

### ISSUE THREAD

**Issue by yifanhunter** [SciYu/HiPhO#2]
I notice that Hugging Face（https://huggingface.co/datasets/SciYu/HiPhO） has only made the training dataset publicly available; when will the test dataset be open-sourced?

**Comment by SciYu** [comment]
Thank you for your interest! HiPhO is designed purely as a benchmark dataset, so the open data on Hugging Face already corresponds to the test set for evaluation. There is no training set in this benchmark.
